docs/4.0/docs/examples/use-private-registry.md

```diff
@@ -0,0 +1,69 @@
+# how to use sealos private registry
+
+## where it is
+
+the sealos private registry running in the first nodes of the cluster,where you create your cluster, you can use the follow comand check it:
+
+```

```

Comment:
Please specify a language for each code blocks. For example:

<pre>
```shell
$ nerdctl ps
```
</pre>

But there's no need to do this when it's not related to a language, like:

<pre>
```
/var/lib/sealos/data/default/rootfs/registry/
```
</pre>